Tasks/NugetPackager/NuGetPackager.ps1

```diff
@@ -44,6 +44,11 @@ if ($b_versionByBuild)
     Write-Verbose "Autoversion: Getting version number from build"
     ##Get Version from Build
     
+    if($Env:SYSTEM_HOSTTYPE -eq "release") {
+        Write-Error (Get-LocalizedString -Key "Autoversion: Getting version number from build option is not supported in releases")
```

Comment:
Interesting - since a release uses a build, I wonder if the build # shouldn't flow into release ... hmmm